packages/api/src/controllers/v1/index.ts

```diff
@@ -78,7 +78,7 @@ export class V1 {
 			redis.del(Keys.Contact.id(contact.id));
 			redis.del(Keys.Contact.email(project.id, contact.email));
 		} else {
-			if (subscribed && contact.subscribed !== subscribed) {
+			if (subscribed !== undefined && contact.subscribed !== subscribed) {
 				contact = await prisma.contact.update({
 					where: { id: contact.id },
 					data: { subscribed },
```

Comment:
Are you sure that Prisma does not complain about subscribed possibly being null here?

Reply:
You are right, made a fix now. If subscribed is null, then defaults to true.

Reply:
Just gave it a quick go myself.

This is what you are looking for, and will work just fine!
```ts
      if (subscribed !== null && contact.subscribed !== subscribed) {
        contact = await prisma.contact.update({where: {id: contact.id}, data: {subscribed}});

        redis.del(Keys.Contact.id(contact.id));
        redis.del(Keys.Contact.email(project.id, contact.email));
      }
```

Reply:
Thank you for your patience. Only my 2nd contribution ever. 

Was struggling and thought it was a bug in my code until I realised nothing I do makes a difference. Haha.

Reply:
Thank you so much for contributing. Don't worry about making mistakes, I make them all the time :)